chromadb/utils/embedding_functions.py

```diff
@@ -232,6 +245,25 @@ def __init__(
         self._api_url = f"https://api-inference.huggingface.co/pipeline/feature-extraction/{model_name}"
         self._session = requests.Session()
         self._session.headers.update({"Authorization": f"Bearer {api_key}"})
+        self._max_token_input_length = None
+        self._model_name = model_name
+
+    def max_token_input_length(self) -> int:
```

Comment:
@HammadB, we don't use the max tokens for HF as we don't have access to tokenisers, and I think we should wait to download them (though we could). So effectively, this is just a way for the user to check the max tokens and validate for themselves as they see fit.